requirements.txt

```diff
@@ -2,7 +2,7 @@
 # extension 'brotlipy' on PyPy
 brotli==1.1.0; platform_python_implementation != "PyPy"
 brotlicffi==1.1.0.0; platform_python_implementation == "PyPy"
-unicodedata2==15.1.0; python_version <= '3.11'
+unicodedata2==17.0.0
```

Comment:
The next Python version (3.15?) will have Unicode 17 data, should we preemptively check for it here?